src/server.js

```diff
@@ -40,6 +42,7 @@ export default class Server {
     app.use('/events', eventsRouter(this.modelEngine.tokenAuthenticator, this.modelEngine.identityManager, this.modelEngine));
     app.use('/token', tokenRouter(this.modelEngine.tokenAuthenticator, this.config));
     app.use('/bundle', bundlesRouter(this.modelEngine));
+    app.use('/health_check', healthCheckHandler(this.mongoClient, this.modelEngine.proofRepository.web3));
```

Comment:
Great idea about health check. I would argue to use RESTfull naming (i.e. simply `/health`).